app/routes/__frontend/resources/virtual-coffee-handbook/get-involved/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+meta:
+  title: Get Involved at Virtual Coffee
+  description: Paths to Leadership and Volunteering at Virtual Coffee.
+hero:
+  Hero: UndrawCollaborators
+tags:
+  - memberresources
+  - memberresourcesIndex
+order: 3
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import FileIndex from '~/components/content/FileIndex';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Our community values creating opportunities for learning, leadership, and contribution for everyone. And there are some ways for you to get involved!
```

Comment:
Our community values creating opportunities for learning, leadership, and contribution for everyone, and there are multiple ways for you to get involved!